wal/wal_other.go

```diff
@@ -0,0 +1,23 @@
+// +build !windows
```

Comment:
add the contents of `$REPO_ROOT/.header` here or it won't pass CI

also keep the build constraint a line separate from the package declaration because it confuses godoc
```
<HEADER>

// +build !windows

package wal
```